python/baseline/utils.py

```diff
@@ -683,37 +683,38 @@ def convert_bio_to_iobes(ifile, ofile):
             if len(tokens) == 0:
                 writer.write('\n')
                 continue
-
-            label = tokens[-1]
-
-            if i + 1 != len(lines):
-                next_tokens = lines[i+1].split()
-                if len(next_tokens) > 1:
-                     next_tag = next_tokens[-1]
-                else:
-                    next_tag = None
-
-            # Nothing to do for label == 'O'
-            if label == 'O':
-                updated_label = label
-
-            # It could be S
-            elif label[0] == 'B':
-                if next_tag and next_tag[0] == 'I' and next_tag[2:] == label[2:]:
+            for field in fields:
+                label = tokens[field]
+
+                if i + 1 != len(lines):
+                    next_tokens = lines[i+1].split()
+                    if len(next_tokens) > 1:
+                         next_tag = next_tokens[-1]
+                    else:
+                        next_tag = None
+
+                # Nothing to do for label == 'O'
+                if label == 'O':
                     updated_label = label
-                else:
```

Comment:
i am not sure where this change is coming from, i cant find this line in master: https://github.com/dpressel/baseline/blob/master/python/baseline/utils.py#L699 and that would make sense.